cfg/rke2-cis-1.24/node.yaml

```diff
@@ -432,6 +432,7 @@ groups:
             - flag: RotateKubeletServerCertificate
               path: '{.featureGates.RotateKubeletServerCertificate}'
               set: false
+        type: manual
```

Comment:
4.2.12 is automated too: https://docs.rke2.io/security/cis_self_assessment124#4212-verify-that-the-rotatekubeletservercertificate-argument-is-set-to-true-automated